graphql_api/actions/repository.py

```diff
@@ -72,16 +70,13 @@ def list_repository_for_owner(
     if exclude_okta_enforced_repos:
         queryset = queryset.exclude_accounts_enforced_okta(okta_account_auths)
 
-    if not ai_enabled_filter:
```

Comment:
no to reviewers: expand the code above this change, and you will see that we early-return `if ai_enabled_filter` above, so this branch here was always true.